src/groupeddataframe/splitapplycombine.jl

```diff
@@ -37,6 +44,14 @@ function _combine_prepare(gd::GroupedDataFrame,
             push!(cs_vec, p)
         end
     end
+    return _combine_prepare_norm(gd, cs_vec, keepkeys, ungroup, copycols,
+                                 keeprows, renamecols)
+end
+
+function _combine_prepare_norm(gd::GroupedDataFrame,
+                               cs_vec::Vector{Any},
```

Comment:
```suggestion
                               cs_vec::AbstractVector,
```

Reply:
I prefer `Vector{Any}` to make sure that in the future we have only one method instance of `_combine_prepare_norm` with respect to this argument (so that we by accident do not introduce compilation latency).